modules/exploits/apple_ios/browser/safari_jit.rb

```diff
@@ -0,0 +1,549 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ManualRanking
+
+  include Msf::Post::File
+  include Msf::Exploit::Remote::HttpServer::HTML
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Safari JIT Exploit',
+        'Description' => %q{
+          This module exploits a JIT optimisation bug in Safari Webkit. This allows us to
```

Comment:
you spelled optimi**z**ation wrong

Reply:
your entire country is spelling optimisation wrong :trollface: